libraries/botbuilder-dialogs-adaptive/src/languageGeneratorExtensions.ts

```diff
@@ -56,7 +61,11 @@ export class LanguageGeneratorExtensions {
             }
         }
 
-        const languageGeneratorManager: LanguageGeneratorManager = new LanguageGeneratorManager(resourceExplorer);
+        let languageGeneratorManager: LanguageGeneratorManager = this._languageGeneratorManagers.get(resourceExplorer);
```

Comment:
Here you can drop the `: LanguageGeneratorManager` piece as well, the type should be properly inferred.

Reply:
Got it, thx.